src/Domain/Entity/Season.php

```diff
@@ -12,6 +13,12 @@ class Season extends Group
     /* @var int */
     private $aggregatedBroadcastsCount;
 
+    /* @var DateTime */
```

Comment:
Just remembered this, but all the other domain models use DateTimeImmutable for date properties to prevent people accidentally/onpurpose changing things in here once it's instantiated. 

Would be better to keep that consistent I think, it's literally a case of replacing the class name here and in the mapper

Reply:
Actually not even the mapper I think, just the typehint, doctrine seems to be already returning the right class.

Reply:
there was a reason I ended up using `DateTime` but I can’t remember it now :) I’ll try again.